make/local.mk

```diff
@@ -0,0 +1 @@
+GCC_REQUIRED_VERSION := 6.3.1
```

Comment:
I suspect this shouldn't be in here, but the file should be listed in `.gitignore`.